barcode-scanning/android/src/main/java/com/rnmlkit/barcodescanning/BarcodeScanningModule.java

```diff
@@ -71,8 +71,9 @@ public void onSuccess(List<Barcode> barcodes) {
                             WritableArray result = Arguments.createArray();
                             for (Barcode barcode : barcodes) {
                                 WritableMap map = Arguments.createMap();
-                                map.putString("value", barcode.getRawValue());
+                                map.putString("value", barcode.getDisplayValue());
```

Comment:
Interesting. So before this change Android was returning the raw value in `value` while iOS was returning the display value in the same field?

Although this is inconsistent behavior, changing the `value` field to always return the display value is a breaking change.

Instead, I'd recommend introducing two new keys `rawValue` and `displayValue` and deprecating the existing `value` property and remove it in a future major release.